### ISSUE TITLE
Openshift `oc get projects` returns error

### ISSUE THREAD

**Issue by gnunn1** [kubernetes-sigs/kui#8928]
**Describe the bug**

When using kui with OpenShift, running the command `oc get projects` returns an error but works fine in a standard terminal
```
$ oc get projects
error: the server doesn't have a resource type "Project"
```

**Steps to reproduce the behavior**

1. Download kui on Linux using the installation instructions on the home page
2. Run Kui
3. Execute `oc get projects` against an OpenShift cluster

**Expected behavior**
A list of OpenShift Projects

**System**

- Kui version: 18.2.3
- Operating system: Arch Linux

**Comment by starpit** [comment]
nice catch!!

**Comment by gnunn1** [comment]
Thanks, BTW just noticed the same issue with `users` and `identity`, i.e.

```
$ oc get users
error: the server doesn't have a resource type "User"
```

```
$ oc get identity
error: the server doesn't have a resource type "Identity"
```

Should I open a new bug report for this?

**Comment by starpit** [comment]
thanks again! i'll reopen.

**Comment by starpit** [comment]
Sorry it took so long to get to this one. The fix will be included in the next release!

**Comment by starpit** [comment]
The fix has been released in 12.0.0. If the problem persists, ping here!
